addons/account/wizard/accrued_orders.py

```diff
@@ -229,9 +226,8 @@ def _ellipsis(string, size):
             total = sum(order.amount_total for order in orders)
             for line in orders.order_line:
                 ratio = line.price_total / total
-                if not is_purchase and line.order_id.analytic_account_id:
-                    account_id = str(line.order_id.analytic_account_id.id)
-                    analytic_distribution.update({account_id: analytic_distribution.get(account_id, 0) +100.0*ratio})
+                if not is_purchase:
+                    self._update_distribution(distribution, order, ratio)
```

Comment:
If you keep the `_update_distribution`, then there is a problem in the arguments here (`distribution` vs `analytic_distribution`, `order`)